src/client.h

```diff
@@ -53,6 +58,20 @@ extern bool                      g_bOnlinePicons;
 extern std::string               g_strOneGroup;
 extern std::string               g_szUserPath;
 extern std::string               g_szClientPath;
+extern int                       g_iEnableTimeshift;
+extern std::string               g_strTimeshiftBufferPath;
+extern int                       g_iReadTimeout;
 extern std::string               g_strChannelDataPath;
 extern ADDON::CHelper_libXBMC_addon *   XBMC;
 extern CHelper_libXBMC_pvr *     PVR;
+
+/*!
+ * @brief PVR macros for string exchange
+ */
+#define PVR_STRCPY(dest, source) do { strncpy(dest, source, sizeof(dest)-1); dest[sizeof(dest)-1] = '\0'; } while(0)
```

Comment:
This is only used in `Signalstatus`. I would not introduce a macro if is only used two times. But this is only my personal POV.